ts/components/PreferencesListItem.tsx

```diff
@@ -40,9 +40,9 @@ const styles = StyleSheet.create({
 export const PreferencesListItem = (props: Props): React.ReactElement => (
   <View style={styles.row} testID={props.testID}>
     <View style={styles.left}>
-      <H4 weight={"SemiBold"} color={"bluegreyDark"}>
+      <H6 weight={"SemiBold"} color={"bluegreyDark"}>
         {props.title}
-      </H4>
+      </H6>
       <H5 weight={"Regular"} color={"bluegrey"}>
```

Comment:
I think it's better to not touch global components.

Reply:
right, reverted 👍